server/models/Version.ts

```diff
@@ -34,7 +35,7 @@ export interface Version {
 
 export type VersionDoc = Version & Document<any, any, Version>
 
-const VersionSchema = new Schema<Version>(
+const VersionSchema: any = new Schema<Version>(
```

Comment:
try VersionSchema: Schema<Version> = new Schema<Version>

Reply:
Covered in BAI-290